src/components/Quiz/QuizzesModal.tsx

```diff
@@ -18,27 +17,28 @@ const QuizzesModal = ({
   onQuizModalClose,
   ...props
 }: QuizzesModalProps) => {
-  const getStatusColor = (): ModalContentProps["bg"] => {
+  // TODO: remove bang in utility class names when Modal is migrated
+  const getStatusColorClass = () => {
     if (quizStatus === "neutral") {
-      return "background.base"
+      return "!bg-background"
```

Comment:
Forcing the styling because these utilities currently compete with Chakra style props from `Modal`